charts/govuk-apps-conf/templates/router-configmap.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: router-nginx-conf
+  labels:
+    {{- include "govuk-apps-conf.labels" . | nindent 4 }}
+    app.kubernetes.io/component: web
+data:
+  nginx.conf: |-
+    user nginx;
+
+    load_module /usr/lib/nginx/modules/ngx_http_perl_module.so;
+
+    error_log  /var/log/nginx/error.log warn;
+    pid        /var/run/nginx.pid;
+
+    events {
+      worker_connections  1024;
+    }
+
+    http {
+      include       /etc/nginx/mime.types;
+      default_type  application/octet-stream;
+
+      server_tokens off;
+
+      sendfile        on;
+      keepalive_timeout  65;
+
+      perl_set $uri_lowercase 'sub {
+        my $r = shift;
+        return lc($r->uri);
+      }';
+
+      upstream router {
+        server 127.0.0.1:3000;
+      }
+
+      server {
+        listen 8080;
+
+        location / {
+          proxy_set_header   Host $http_host;
+          proxy_set_header   X-Real-IP $remote_addr;
+          proxy_pass         http://router;
+          proxy_redirect     off;
+        }
+        location /assets {
+          proxy_set_header   Authorization "";
+          proxy_set_header   Connection "";
+          proxy_set_header   X-Real-IP $remote_addr;  # TODO: pass the actual end-client address
+          proxy_hide_header  x-amz-id-2;
+          proxy_hide_header  x-amz-meta-server-side-encryption;
+          proxy_hide_header  x-amz-request-id;
+          proxy_hide_header  x-amz-server-side-encryption;
+          proxy_hide_header  x-amz-version-id;
+          add_header         Cache-Control max-age=31536000;
+          proxy_intercept_errors on;
+          proxy_pass         https://govuk-app-assets-{{ .Values.govukEnvironment }}.s3.eu-west-1.amazonaws.com;
+        }
+        location = /__canary__ {
+          return 200 '{"message": "Tweet tweet"}';
+        }
+        location ~ ^\/[A-Z]+[A-Z\W\d]+$ {
+          rewrite ^(.*)$ $scheme://$host$uri_lowercase permanent;
+        }
+        location = /robots.txt {
+          root /usr/share/nginx/html;
+        }
+      }
+    }
+  robots.txt: |-
+    {{- if eq .Values.govukEnvironment "staging" }}
```

Comment:
I would think we could have Production as the base case and all other cases (non-live environments like staging, integration, test, etc) disallow all robots.